benchmarks/experiment_runner.py

```diff
@@ -906,6 +932,11 @@ def main():
     logger.warning('Enabling fast F32 multiplication for PyTorch')
     torch.set_float32_matmul_precision('high')
 
+  if args.profile_xla:
+    logger.info(
+        'Enabling XLA XPlane profiling. Do not benchmark with this option set.')
+    server = xp.start_server(9012)
```

Comment:
consider adding a FLAGS.profiler_xla_port default to 9012

Reply:
Do you mean importing some specific constant, or parametrising server port?

Reply:
I meant parametrising server port, such as https://github.com/pytorch/xla/blob/dc08cb6893e84f29fc31ed4e44e8998ed680b2a1/test/spmd/test_train_spmd_imagenet.py#L380